setup/mail-postfix.sh

```diff
@@ -198,7 +207,8 @@ tools/editconf.py /etc/postfix/main.cf lmtp_destination_recipient_limit=1
 # "450 4.7.1 Client host rejected: Service unavailable". This is a retry code, so the mail doesn't properly bounce. #NODOC
 tools/editconf.py /etc/postfix/main.cf \
 	smtpd_sender_restrictions="reject_non_fqdn_sender,reject_unknown_sender_domain,reject_authenticated_sender_login_mismatch,reject_rhsbl_sender dbl.spamhaus.org" \
```

Comment:
Seems strange that it's only the `smtpd_recipient_restrictions=` line that changes but not the `smtpd_sender_restrictions=` line?

Reply:
The smtpd_sender_resctrictions apply to email sent from the box.  This pull request only deals with whether a message will be received (which is smtpd_recipient_restrictions).

It checks an incoming email message to see whether the sender matches a regular expression or whether the email's recipient matches a regular expression.

Reply:
Thanks.

Reply:
Thanks. I remember now why I never liked the config names I found in postfix. :-/

Reply:
I have to agree, they are not the clearest and at first I thought the same as you about smtpd_sender_restrictions.